lib/content-services/src/lib/category/categories-management/categories-management.component.spec.ts

```diff
@@ -0,0 +1,487 @@
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Category, CategoryPaging, ResultNode, ResultSetPaging } from '@alfresco/js-api';
+import { DebugElement } from '@angular/core';
+import { ComponentFixture, discardPeriodicTasks, fakeAsync, flush, TestBed, tick } from '@angular/core/testing';
+import { Validators } from '@angular/forms';
+import { MatError } from '@angular/material/form-field';
+import { MatListOption, MatSelectionList } from '@angular/material/list';
+import { By } from '@angular/platform-browser';
+import { TranslateModule } from '@ngx-translate/core';
+import { of, Subject } from 'rxjs';
+import { ContentTestingModule } from '../../testing/content.testing.module';
+import { CategoriesManagementMode } from '../categories-management-mode';
+import { CategoryService } from '../services/category.service';
+import { CategoriesManagementComponent } from './categories-management.component';
+
+describe('CategoriesManagementComponent', () => {
+    let component: CategoriesManagementComponent;
+    let fixture: ComponentFixture<CategoriesManagementComponent>;
+    let categoryService: CategoryService;
+    const classifiableChangedSubject = new Subject<void>();
+    const category1 = new Category({ id: 'test', name: 'testCat' });
+    const category2 = new Category({ id: 'test2', name: 'testCat2' });
+    const category3 = new Category({ id: 'test3', name: 'testCat3' });
+    const category4 = new Category({ id: 'test4', name: 'testCat4' });
+    const resultCat1 = new ResultNode({ id: 'test', name: 'testCat', path: { name: 'general/categories' }});
+    const resultCat2 = new ResultNode({ id: 'test2', name: 'testCat2', path: { name: 'general/categories' }});
+    const categoryPagingResponse: CategoryPaging = { list: { pagination: {}, entries: [ { entry: category1 }, { entry: category2 }]}};
+    const categorySearchResponse: ResultSetPaging = { list: { pagination: {}, entries: [ { entry: resultCat1 }, { entry: resultCat2 }]}};
+
+    beforeEach(() => {
+        TestBed.configureTestingModule({
+            declarations: [CategoriesManagementComponent],
+            imports: [
+                TranslateModule.forRoot(),
+                ContentTestingModule
+            ],
+            providers: [
+                {
+                    provide: CategoryService,
+                    useValue: {
+                        getSubcategories: () => of(categoryPagingResponse),
+                        searchCategories: () => of(categorySearchResponse)
+                    }
+                }
+            ],
+            teardown: { destroyAfterEach: true }
+        });
+
+        fixture = TestBed.createComponent(CategoriesManagementComponent);
+        component = fixture.componentInstance;
+        categoryService = TestBed.inject(CategoryService);
+    });
+
+    function getNoCategoriesMessage(): HTMLParagraphElement {
```

Comment:
All usages looks like: getNoCategoriesMessage().textContent.trim() so you can just return string directly to don't repeat whole chain